src/Agent/MsiInstaller/Installer/Product.wxs

```diff
@@ -809,13 +810,19 @@ SPDX-License-Identifier: Apache-2.0
           <MultiStringValue>COR_ENABLE_PROFILING=1</MultiStringValue>
           <MultiStringValue>COR_PROFILER={71DA0A04-7777-4EC6-9643-7D28B46A8A41}</MultiStringValue>
           <MultiStringValue>NEWRELIC_INSTALL_PATH=[NETAGENTFOLDER]</MultiStringValue>
+          <?ifdef IsWin64?>
+            <MultiStringValue>COR_PROFILER_PATH_32=[NETAGENT32FOLDER]NewRelic.Profiler.dll</MultiStringValue>
```

Comment:
You've verified that it works correctly if you have `COR_PROFILER_PATH` and `COR_PROFILER_PATH_32` set but not `COR_PROFILER_PATH_64`? I think it should, but the documentation is a little vague on how the priorities work.

Reply:
I read the docs and it looks like ti will work, but I will test.

Reply:
Yes, it works.

Reply:
Great, thanks!